app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,40 @@
+<resources>
+    <string name="main_activity_description">Chat with your mentors.</string>
+
+    <string name="login">Login</string>
+    <string name="chat_with_mentor">Chat with a mentor</string>
+
+    <!-- NOTIFICATION SERVICE -->
+    <string name="notification_title">You have a new notification</string>
+
+    <!-- MAIN ACTIVITY -->
+    <string name="error_not_connected">You must first login!</string>
+    <string name="error_authentication_canceled">Login aborted</string>
+    <string name="error_no_internet">No internet connection!</string>
+    <string name="error_unknown_error">An unknown error occurred</string>
+    <string name="connection_succeed">Login successful!</string>
+    <string name="button_login_text_logged">View your profile</string>
+    <string name="button_login_text_not_logged">Login</string>
+
+    <!-- LOGIN ACTIVITY -->
+    <string name="toolbar_title_login_activity">Your Profile</string>
+    <string name="text_input_username">Username</string>
+    <string name="text_view_email">Email address:</string>
+    <string name="button_update_account">Update username</string>
+    <string name="button_delete_account">Delete my account</string>
+    <string name="button_sign_out_account">Logout</string>
+    <string name="info_no_email_found">No email address found</string>
+    <string name="info_no_username_found">No username found</string>
+    <string name="popup_message_confirmation_delete_account">Are you sure you wish to delete your account?</string>
+    <string name="popup_message_choice_yes">Yes!</string>
+    <string name="popup_message_choice_no">No</string>
+    <string name="check_box_is_mentor">Are you a mentor?</string>
+
+    <!-- MENTOR_CHAT ACTIVITY -->
+    <string name="toolbar_title_mentor_chat_activity">Discussion Chats</string>
```

Comment:
Delete discussion and just leave "Chats", "Chats de discussion" is objectively bizarre in French (and it should be "Tchats", "Chats" is technically "cats"), I don't know what happened there.

Reply:
I think this was to signify the difference between the chat_rooms_ and chat _messages_...

I'm thinking Chat Rooms